Makefile

```diff
@@ -373,15 +373,20 @@ test-watch-for-repl: ##@test Watch all Clojure tests and support REPL connection
 		"until [ -f $$SHADOW_OUTPUT_TO ] ; do sleep 1 ; done ; node --require ./test-resources/override.js $$SHADOW_OUTPUT_TO --repl"
 
 test-unit: export SHADOW_OUTPUT_TO := target/unit_test/test.js
-test-unit: export SHADOW_NS_REGEXP := ^(?!status-im\.integration-test).*-test$$
+test-unit: export SHADOW_NS_REGEXP := ^(?!tests\.integration-test|?!tests-im\.contract-test).*-test$$
```

Comment:
hmm I think this regex needs work 🤔

Reply:
```suggestion
test-unit: export SHADOW_NS_REGEXP := ^(?!tests\.integration-test)(?!tests-im\.contract-test).*-test$$
```

Reply:
For me it's fine, @ilmotta any particular read it was added this way? (I just copied over what was used for integration tests)

Reply:
oh actually maybe I added this one 😅 I'll try!

Reply:
added and the tests pass locally.